src/sonic-config-engine/minigraph.py

```diff
@@ -1880,6 +1880,13 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     for port in ports.values():
         port['mtu'] = '9100'
         port['tpid'] = '0x8100'
+    # mode check for vlan membership in PORT table default mode is routed and for vlan membership it is trunk
```

Comment:
># mode check for vlan membership in PORT table default mode is routed and for vlan membership it is trunk [](http://example.com/codeflow?start=4&length=105)

Since you can implement a conditional default value in relevant consuming component, no need to add explicit attribute  during load_minigraph. We try to prevent any minigraph.py behavior change now.

Reply:
Here in minigraph .py, we implemented conditional default value in relevant consuming component. 
Do we need to remove all this conditional change as well from minigraph.py